src/network-config.js

```diff
@@ -66,6 +66,25 @@ export const networkConfigs = {
       live: true,
     },
   },
+  [chains.getChainInformation(10).type]: {
+    isActive: true,
+    addresses: {
+      ensRegistry:
+        localEnsRegistryAddress || '0x6f2CA655f58d5fb94A08460aC19A552EB19909FD',
+      dai: '0xdc31ee1784292379fbb2964b3b9c4124d8f89c60',
+      governExecutorProxy: null,
+    },
+    nodes: {
+      defaultEth: null, // TBD
```

Comment:
Please add any public rpc. We won't provide optimism endpoints for now
For example `https://optimism.llamarpc.com`